third_party/silabs/SiWx917_sdk.gni

```diff
@@ -103,15 +138,52 @@ template("siwx917_sdk") {
       "USE_NVM3=1",
       "RSI_ARM_CM4F",
       "SIWX_917",
-      "BRD4325A",
       "CHIP_9117",
-      "RSI_BLE_ENABLE=1",
+      "BLE_ENABLE=1",
+      "RSI_LITTLE_ENDIAN=1",
       "TINYCRYPT_PRIMITIVES",
       "OPTIMIZE_TINYCRYPT_ASM",
       "__error_t_defined",
       "RSI_SAMPLE_HAL",
+      "ENABLE_IPMU_APIS",
+      "ROM_BYPASS",
+      "SI917",
+      "DEBUG_UART",
+      "FLASH_BASED_EXECUTION_ENABLE",
+      "TA_DEEP_SLEEP_COMMON_FLASH",
+      "SL_WIFI_COMPONENT_INCLUDED",
+      "CHIP_917",
+      "ROMDRIVER_PRESENT",
+      "SL_CATALOG_FREERTOS_KERNEL_PRESENT=1",
+      "SL_PLATFORM_EXAMPLES_ENABLE",
+      "SI91X_DEVICE",
+      "SI917_RADIO_BOARD",
+      "FLASH_PAGE_SIZE",
+      "SL_NVM3_PRESENT",
+      "ROM_WIRELESS",
+      "SRAM_BASE=0x0C000000UL",
+      "SRAM_SIZE=0x00040000UL",
     ]
 
+    if (silabs_board == "BRD4325B") {
+      defines += [
+        "BRD4325A",
+        "BRD4325B",
+        "DUAL_FLASH_EN",
+      ]
+    }
+    if (silabs_board == "BRD4325C") {
+      defines += [
+        "BRD4325C",
+        "CHIP_917 = 1",
```

Comment:
Why are we duplicating the define again ?
And Why do we also have the CHIP_917B0 ? yet another define for the 917 ?

Reply:
We common flash is a BO board and dual flash is A0 board.

It is their terminology these defines were used in their SDK to distinguish common and dual flash